gatsby-theme-portfolio-minimal/src/components/ArticleCard/index.tsx

```diff
@@ -38,6 +38,7 @@ export function ArticleCard(props: ArticleCardProps): React.ReactElement {
                         <GatsbyImage
                             className={classes.ImageWrapper}
                             imgClassName={classes.Image}
+                            objectFit={props.data.image.objectFit || 'cover'}
```

Comment:
Can we remove the fallback `|| 'cover'`?

So you are able to specify no "object-fit" at all if you don't want to